README.md

```diff
@@ -14,72 +15,118 @@ OSPSuite.ReportingEngine implementation in R
 
 <!-- badges: end -->
 
-# Installation
+## Installation
 
-The **OSPSuite.ReportingEngine** package is compatible with version 4.x.x of R. Please follow the installation instructions below:
+The `{ospsuite.reportingengine}` package is compatible with version 4.x.x of R. Please follow the installation instructions below:
 
-**OSPSuite.ReportingEngine** requires following packages to be installed:
+### From Github <img src=https://avatars.githubusercontent.com/github width=15px></img>
 
-- From CRAN:
-  - **ggplot2** (>= 3.3.0)
-  - **jsonlite**
-  - **knitr**
-  - **pander**
-  - **patchwork**
-  - **readxl**
-  - **reshape2**
-  - **rmarkdown**
-  - **styler**
-  - ... (s. [Installation instructions for the package **ospsuite**](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation))
-- Must be downloaded manually:
-  - **ospsuite.utils** (>= 1.4.0)
-    - [Package download](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases)
-  - **tlf** (>= 1.5.0)
-    - [Package download](https://github.com/Open-Systems-Pharmacology/TLF-Library/releases)
-  - **rSharp** (s. [Installation instructions for the package **ospsuite**](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation))
-  - **ospsuite** (>= 11.1)
-    - [Package download](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases)
-    - [Installation instructions](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation)
+You can install the development version of `{ospsuite.reportingengine}` from [GitHub](https://github.com/) with:
 
-[OPTIONAL] Install **Pandoc** (required for generation of reports in MS-Word format):
+```r
+# install.packages("remotes")
+remotes::install_github("Open-Systems-Pharmacology/OSPSuite.ReportingEngine")
+```
+### Using the package bundle &#128230;
 
-* [Pandoc Installer (Windows)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-windows-x86_64.msi)
-* [Pandoc Installer (Linux)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-linux-amd64.tar.gz)
+You can install a specific version of the `{ospsuite.reportingengine}` package by downloading and installing its zip or tar.gz bundle [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/history).

```

Comment:
would move this part as the last bullet point in the list below. 
And also set the release version as the first bullet point in the list. So
* Release
* Development
* Specific version